src/RacingController.js

```diff
@@ -1,48 +1,48 @@
-import InputView from './InputView';
-import Validation from './utils/Validation';
-import OutputView from './OutputView';
-import Cars from './Cars';
+import InputView from './view/InputView';
+import OutputView from './view/OutputView';
+import Cars from './domain/Cars';
 
 export class Game {
   async play() {
-    const carNameArray = await Game.getCarNames();
+    const carNameArray = await this.getCarNames();
     const cars = new Cars(carNameArray);
-    const tryCount = await Game.getTryCount();
-
-    Game.moveCars(cars, tryCount);
+    const tryCount = await this.getTryCount();
+    OutputView.printResultTitle();
```

Comment:
정말 간단한 함수(e.g. 2줄 짜리)가 아니면 가독성을 위해 선언부와 실행부 사이에 개행을 두면 좋아요.

```suggestion
    const tryCount = await this.getTryCount();
    
    OutputView.printResultTitle();
```

Reply:
이것도 저번 리뷰 때 받았던 피드백이었는데 놓쳤습니다 ㅠㅠ
꼼꼼하게 확인하도록 할게요 🥲